WebApps/CNS.Auth.Web/Services/CNSCertificateService.cs

```diff
@@ -136,10 +156,17 @@ public async Task<X509Certificate2Collection> GetTrustedCertificateCollection(Ca
 				.Select(x => new X509Certificate2(Convert.FromBase64String(x.Value)));
 
 			return new X509Certificate2Collection(certificates.ToArray());
-			
+
 		}
 
-		public async Task GetCertificateAuthenticationOptions(CertificateAuthenticationOptions options,CancellationToken cancellationToken = default)
+
+		/// <summary>
+		/// Get certificate authentication options by CertificateAuthenticationOptions object
+		/// </summary>
+		/// <param name="options">Option used to configure certificate authentication</param>
+		/// <param name="cancellationToken">Cancellation Token</param>
+		/// <returns></returns>
+		public async Task GetCertificateAuthenticationOptions(CertificateAuthenticationOptions options, CancellationToken cancellationToken = default)
```

Comment:
Maybe the method name is misleading, since this method doesn't "get" the options, it just configures them. For instance, the method doesn't return anything, it's just a Task and takes the options as an input parameter. Can you rename the method in the implementation and in the interface to match something like "ConfigureCertificateAuthenticationOptions"? As a consequence, please update also the inline documentation of the method.